internal/internal_command_state_machine.go

```diff
@@ -164,6 +164,12 @@ type (
 	stateMachineIllegalStatePanic struct {
 		message string
 	}
+
+	// Error returned when a child workflow with the same id already exists and hasn't completed
+	// and been removed from internal state.
+	childWorkflowExistsWithId struct {
```

Comment:
Why create a new error instead of using `ChildWorkflowExecutionAlreadyStartedError`?

Reply:
That's a proto - so it doesn't impl the `Error` interface and feels icky to use in this place